src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	/**
+	 * @return the feedbackToUser
+	 */
+	public String getFeedbackToUser() {
+		return feedbackToUser;
```

Comment:
I believe the indentation at line 40 is a tab.
Do be careful.
It could be due to eclipse's profile bug that caused your 4 spaces indentation to revert back to tab.